examples/subscription_handler_example.py

```
@@ -0,0 +1,668 @@
+
+5. Watch the server logs to see callback messages being sent:
+   - KEEP_ALIVE messages every 5 seconds
+   - DATA messages with counter values
+   - COMPLETE message when the subscription ends
+"""
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Ruff (0.15.1)

[error] Possible binding to all interfaces

(S104)